MANIFESTO.md

```diff
@@ -0,0 +1,27 @@
+
+##  The Apollo manifesto: Open Capability--Shared Resources--Accelerated Innovation--Sustained Mutual Benefit.
+
+In essence, Apollo is the freedom from wasting time and resources re-implementing fundamental components and allows each member of the ecosystem to focus on their specific areas of expertise, which will dramatically increase the speed of innovation.
+
+Apollo’s comprehensive, modular solution allows anyone to use as much or as little of the existing source code and data. Open source code portions can be modified and open capability components accessible through an API can be replaced with proprietary implementations. All this can then be contributed back to Apollo, redistributed, and commercialized.
+
+Critical considerations such as ensuring reliability, handling failover, and guaranteeing security that would pose a challenge to most development efforts will be readily available components. Everyone can benefit from Apollo's capabilities to improve individual competitive advantage.
+
+Apollo provides high quality code and data that allows anyone to bootstrap their autonomous driving development, but only with contributions from Apollo’s partners and the wider community can Apollo become increasingly more capable. The goal is to create a virtuous cycle where software and services are deployed onto the vehicle to obtain quality data, which is then used to create an even more capable autonomous system. Accelerating this iterative cycle is the engine of innovation that will give birth to a viable autonomous vehicle and compelling services leading to a breakthrough autonomous ecosystem.
+
+Apollo welcomes anyone to make technical and non-technical contributions, but the most valuable contribution is high quality data, it is the very fuel that drives innovation. Baidu will seed Apollo with its vast dataset that the community can experiment with. However, Baidu’s contribution alone is not enough to sustain Apollo’s evolution let alone realize Apollo’s grand ambition. Therefore, Apollo will create an innovative data collective that will pool high quality data from Apollo’s partners, each of whom can leverage a larger data set than it can gather individually.
+
+This innovative platform and data service creates a lasting, mutually beneficial business model that accelerates individual innovation as well as sustains the growth of a breakthrough ecosystem. On a broader level, Apollo will engage academia, government, and the general public in order to convince and educate society on the acceptance and adoption of autonomous technology. Only by growing the ecosystem together rather than through individual efforts can the full potential of autonomous vehicles and services be realized for the benefit of all.
+
+## Attribution
+
+This Manifesto is available at <http://apollo.auto/docs/manifesto.html>
```

Comment:
This Manifesto is available on our website [apollo.auto](http://apollo.auto/docs/manifesto.html)